Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m

```diff
@@ -115,11 +119,8 @@ - (void)configureWithConversationViewItem:(ConversationViewItem *)viewItem isOve
             OWSAssert(statusIndicatorImage.size.width <= self.maxImageWidth);
             self.statusIndicatorImageView.image =
                 [statusIndicatorImage imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
-            if (messageStatus == MessageReceiptStatusRead) {
-                self.statusIndicatorImageView.tintColor = [UIColor ows_signalBlueColor];
-            } else {
-                self.statusIndicatorImageView.tintColor = textColor;
-            }
+            // TODO: How to indicate read status for transitional colors?
+            self.statusIndicatorImageView.tintColor = textColor;
```

Comment:
Why are we getting rid of the "read" color hear?

I see your comment - but I'm not sure I understand what you mean by "transitional colors". Do you mean while something is uploading?

AFAIK we're keeping blue for the read state.

Reply:
I've talked to Myles and settled a bunch of TODOs in this PR. This comment can now be removed.

Reply:
> AFAIK we're keeping blue for the read state.

It doesn't appear so.